<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompanyDetailDialog</class>
 <widget class="QDialog" name="CompanyDetailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_Name">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_Cash">
     <property name="text">
      <string>Cash</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_TotalValue">
     <property name="text">
      <string>TotalValue</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_Liability">
     <property name="text">
      <string>Liability</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
